Training/box_it.cpp

```diff
@@ -1,100 +1,108 @@
 #include <iostream>
 #include "box_it.h"
-
 using namespace std;
 
 class Box {
 private:
-    int m_l = 0,  // length
-        m_b = 0,  // breadth
-        m_h = 0;  // height
+	int m_l = 0;  // length
```

Comment:
Is there a reason to change these to be independent statements?

Reply:
Style only--

When people are interacting with your code or you're making edits or thinking about things, being able to comment a line and not break anything else is very nice.

There are quite a few styles that are built entirely around making your code accessible to single line comments. This is more important in C++, because maintainability becomes an issue over time.

I'll regularly introduce some examples of this here and there.